### PR TITLE
ref(osm/*): rename osm mesh delete to osm mesh uninstall

### LINKED ISSUE #1622: rename osm mesh delete to osm mesh uninstall
"uninstall" seems a bit clearer and this is the terminology Helm uses for removing Kubernetes resources as well. Thoughts?

### PR DESCRIPTION
**Pull request by shalier**
Rename to more accurately describe command
and resolves #1622

Please mark with X for applicable areas.

- New Functionality      [ ]
- Documentation          [X ]
- Install                [ ]
- Control Plane          [ ]
- CLI Tool               [X ]
- Certificate Management [ ]
- Networking             [ ]
- Metrics                [ ]
- SMI Policy             [ ]
- Security               [ ]
- Tests / CI System      [ ]
- Other                  [ ]

Please answer the following questions with yes/no.

- Does this change contain code from or inspired by another project? If so, did you notify the maintainers and provide attribution?
No

### CODE REVIEW

**Review by draychev — COMMENTED** [review]
@shalier Thank you for the change!

You can run `go fmt ./...` to auto-format all files in the repo and push another commit - this will fix the lint issue and pass CI.

Also I'd reccomend using `git mv cmd/cli/mesh_delete.go cmd/cli/mesh_uninstall.go`, which will preserve the history of the file (right now it looks like delete + add)

**Comment by shashankram** [comment]
> Also I'd reccomend using `git mv cmd/cli/mesh_delete.go cmd/cli/mesh_uninstall.go`, which will preserve the history of the file (right now it looks like delete + add)

Please use `git mv` if it isn't the case to preserve history.

File: cmd/cli/mesh_uninstall.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"helm.sh/helm/v3/pkg/action"
+	helmStorage "helm.sh/helm/v3/pkg/storage/driver"
+)
+
+const meshUninstallDescription = `
+This command will uninstall an instance of the osm control plane
+given the mesh name and namespace. It will not uninstall the namespace
```
Comment (ksubrmnn) [inline cmd/cli/mesh_uninstall.go:15]: This should still say "delete the namespace"

**Review by ksubrmnn — CHANGES_REQUESTED** [review]

**Review by ksubrmnn — APPROVED** [review]

**Review by shashankram — APPROVED** [review]

**Comment by michelleN** [comment]
@shalier - thanks for the change. Please remember to hit "Squash and Merge" when you have an approved pull request or are merging community pull requests.

**Pull request merged**